.github/ISSUE_TEMPLATE/Bug_report.md

```diff
@@ -1,44 +1,44 @@
----

```

Comment:
These changes are a bit of a side-effect of another auto-review i was experimenting with, as these files used the wrong line-ending.